### PR TITLE
new recipe: metalign

### PR DESCRIPTION
**Pull request by nlapier2**
Adds [Metalign](https://github.com/nlapier2/Metalign), addresses [nlapier2/Metalign#20](https://github.com/nlapier2/Metalign/issues/20)

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/metalign/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/nlapier2/Metalign/archive/v{{ version }}.tar.gz
+  sha256: 15ff4ce2aeb015fbfc6b01c8baf4e1fa75ed43c45467c795dd7df364899cce16
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (dkoslicki) [inline recipes/metalign/meta.yaml:14]: @nlapier2 This is probably the line causing the Circle CI error. You’ll need to either:
1. Create a `setup.py` in Metalign or
2. Find an alternate install pip install command

I assume 1 would be easier due to Metalign not having any dependencies besides KMC, minimap2 and CMash, so the setup.py would be straightforward. I.e. Create a minimal setup.py as if you intend to release on PyPi

**Comment by dkoslicki** [comment]
@nlapier2 It appears that the `pip install` command is failing due to:
```
00:36:38 BIOCONDA INFO (OUT) ERROR: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.
```
This is most likely due to Metalign not having a `setup.py` file. See [my comment](https://github.com/bioconda/bioconda-recipes/pull/21711#discussion_r413462234) and try running this command locally in your (fresh) Metalign directory and conda env to see if the `setup.py` is correctly formatted. An example (which contains way more extra fluff than you will need) is included [here](https://github.com/dkoslicki/CMash/blob/master/setup.py) with other minimal `setup.py` scripts searchable via google.

**Comment by dkoslicki** [comment]
Accidentally created a new branch in a53e722 instead of directly to @nlapier2's PR. Asking for permission from him to make changes directly to this PR

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

@dkoslicki You can merge them too :)

**Pull request merged**